src/components/layout/CyVerseAppBar.js

```diff
@@ -642,104 +613,81 @@ function CyverseAppBar(props) {
+                        </IconButton>
                     </div>
-                    <Divider />
-                    {drawerItems}
-                    {open && adminUser && (
-                        <>
-                            <Divider />
-                            {adminDrawerItems}
-                        </>
-                    )}
-                </Drawer>
-            </Hidden>
-            <Hidden only={["sm", "md", "lg", "xl"]}>
-                <Drawer
-                    className={clsx(classes.drawer, {
-                        [classes.drawerOpen]: open,
-                        [classes.drawerClose]: !open,
-                    })}
-                    classes={{
-                        paper: clsx({
-                            [classes.drawerOpen]: open,
-                            [classes.drawerClose]: !open,
-                        }),
-                    }}
-                    open={open}
-                    onClose={toggleDrawer(false)}
-                >
+                </Hidden>
+                <Hidden smUp>
                     <div
                         id={build(ids.DRAWER_MENU, ids.ACCOUNT_MI)}
                         style={{ margin: 8 }}
                     >
-                        <UserMenu
-                            baseId={build(ids.DRAWER_MENU, ids.ACCOUNT_MI)}
-                            profile={userProfile}
-                            onLogoutClick={onLogoutClick}
-                            onManageAccountClick={() =>
-                                window.open(
-                                    "https://user.cyverse.org",
-                                    "_blank"
-                                )
-                            }
-                        />
+                        {userProfile ? (
+                            <UserMenu
+                                baseId={build(ids.DRAWER_MENU, ids.ACCOUNT_MI)}
+                                profile={userProfile}
+                                onLogoutClick={onLogoutClick}
+                                onManageAccountClick={() =>
+                                    window.open(
+                                        constants.CYVERSE_USER_PORTAL,
+                                        "_blank"
+                                    )
+                                }
+                            />
+                        ) : (
+                            accountAvatar
+                        )}
                     </div>
-                    <Divider />
-                    {drawerItems}
-                    {adminUser && (
-                        <>
-                            <Divider />
-                            {adminDrawerItems}
-                        </>
-                    )}
-                </Drawer>
-            </Hidden>
+                </Hidden>
+                <Divider />
+                {drawerItems}
+                {(!isXsDown || open) && adminUser && (
```

Comment:
I realize this line is probably why those admin items are showing up where they shouldn't, heh.